packages/eslint-config/index.cjs

```diff
@@ -32,7 +32,6 @@ module.exports = {
     'default-param-last': 'error',
     'guard-for-in': 'error',
     'no-array-constructor': 'error',
-    'no-div-regex': 'error',
```

Comment:
This lint error was triggered on this line:

```ts
.replace(/={1,2}$/, ''); // Remove trailing =
```

Which I guess doesn't bother me.  We could keep it in our ESLint config, and add a single line override if anyone cares.